apps/web/app/(app)/environments/[environmentId]/surveys/components/SurveyList.tsx

```diff
@@ -134,67 +122,45 @@ export const SurveysList = ({
     setSurveys(newSurveys);
   };
 
-  const handleDuplicateSurvey = async (survey: TSurvey) => {
+  const handleDuplicateSurvey = async (survey: TSurveyListItem) => {
```

Comment:
why do we pass down this `duplicateSurvey` so deep? Can you replace that with a router.refresh() after the survey has been duplicated?

Reply:
`router.refresh()` wont work with this setup because we are using Client-Side Data Fetching with useEffect , or we would need to switch to fetching data in server component (page.tsx) using `filters` and `page` numbers as search params 🤔